README.md

```diff
@@ -0,0 +1,126 @@
+# Complete integration sample for the Corbado web component
+This is a sample implementation of frontend and backend where the Corbado web component is integrated. You can see the live demo [here](TODO: Link einfügen wenn online)
+
+>**Warning**
+>In this tutorial a customer system is built with no preexisting user base, which is effecively realized by having the loginInfo endpoint always return a 404 code when given a username. In case you have an existing user base, you just need to modify this endpoint to return other status codes as well, as described in our [docs](TODO: Link einfügen wenn online)
```

Comment:
wir haben doch einen extra Abschnitt für "no existing user base" in the docs. Was ist mit dem?